2.3/test/test_s2i.py

```diff
@@ -0,0 +1 @@
+../../2.4/test/test_s2i.py
```

Comment:
This is, I'd say, why we have `container-common-scripts` repo. I'd rather not symlink between versions.

Reply:
Or just put it into `$ROOT/test` and symlink from the versioned directories.